src/style.css

```diff
@@ -0,0 +1,131 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
```

Comment:
remove padding from each element inside the card and add padding for all sides for container